python/requirements/ml/requirements_dl.txt

```
@@ -4,12 +4,13 @@
 tensorflow==2.6.2
 tensorflow-probability==0.14.1
 
-# If you make changes to the torch versions below, please also make the corresponding changes to `requirements_ml_docker.txt`!
+# If you make changes to the versions below, please also make the corresponding changes to `requirements_ml_docker.txt`!
 
-torch==1.9.0;sys_platform=="darwin"
-torchvision==0.10.0;sys_platform=="darwin"
-
-# On non-OSX machines only install CPU version of torch and torchvision
--f https://download.pytorch.org/whl/torch_stable.html
-torch==1.9.0+cpu;sys_platform!="darwin"
-torchvision==0.10.0+cpu;sys_platform!="darwin"
+--extra-index-url https://download.pytorch.org/whl/cpu  # for CPU versions of torch, torchvision
+--find-links https://data.pyg.org/whl/torch-1.9.0+cu111.html  # for GPU versions of torch-scatter, torch-sparse ref https://github.com/ray-project/ray/pull/26072#discussion_r929330581
+torch==1.9.0
+torchvision==0.10.0
+torch-scatter==2.0.9
+torch-sparse==0.6.12
+torch-geometric==2.0.3; python_version == '3.6'
+torch-geometric==2.0.4; python_version >= '3.7'
```

@amogkam from their [docs](https://pytorch-geometric.readthedocs.io/en/2.0.4/notes/installation.html), should we also add these?

```suggestion
torch-cluster==1.5.9
torch-spline-conv==1.2.1
torch-geometric==2.0.3; python_version == '3.6'
torch-geometric==2.0.4; python_version >= '3.7'
```